_sass/_uswds-custom-styles.scss

```diff
@@ -0,0 +1,164 @@
+    }
+
+    // Menu links on mobile and desktop
+    a,
+    button {
+      color: color("white");
+
+      &:hover {
+        color: color("white");
+        background-color: $c-dhs-blue-60;
+      }
+
+      @include at-media($theme-header-min-width) {
+        &:hover {
+          background-color: transparent;
+        }
+      }
+    }
+
+    // Need to override USWDS.
+    .usa-current {
+      color: color("white");
+
+      // Border on current link
+      &::after {
+        background-color: currentColor !important;
+        border-radius: radius(0);
+      }
+    }
+
+    // The border on the rest of the primary links on desktop hover.
+    .usa-nav__link:hover::after {
+      @include at-media($theme-header-min-width) {
+        background-color: currentColor !important;
+      }
+    }
+
+    // Main menu buttons
+    button[aria-expanded="false"] {
+      // This mixin allows us to define our own path for our custom images.
+      @include add-background-svg("plus-white", "../../img");
+
+      @include at-media($theme-header-min-width) {
+        &,
+        &:hover {
+          @include add-background-svg("angle-arrow-down-white");
+        }
+      }
+    }
+
+    button[aria-expanded="true"] {
+      @include add-background-svg("minus-white", "../../img");
+
+      @include at-media($theme-header-min-width) {
+        background-color: $c-dhs-light-blue-70;
+        @include add-background-svg("angle-arrow-up-white");
+      }
+    }
+  }
+
+  &__submenu-item {
+    border-top-color: $c-dhs-blue-40;
+  }
+}
+
+//
+// Secondary Nav (Search)
+// ==========================================================================
+
+// Remove these if you're adding secondary links.
+.usa-nav__secondary {
+  @include at-media($theme-header-min-width) {
+    bottom: px-to-rem(80);
+    margin-top: 0;
+  }
+
+  .usa-search {
+    @include at-media($theme-header-min-width) {
+      margin-top: 0;
+    }
+  }
+}
+
+
+// ==========================================================================
+// Hero
+// ==========================================================================
+.usa-hero {
+
+  &__callout {
+    background-color: $c-dhs-blue;
+  }
+}
+
+// ==========================================================================
+// Cards
+// ==========================================================================
+// Seen on homepage. A card consisting of header on a blue background and media.
+.usa-card-group--primary-bg {
+
+  // Generic styles
+  a {
+    color: color("white");
+    text-decoration: none;
+  }
+
+  // Component elements
+  .usa-card__container {
+    background-color: $c-dhs-blue;
+  }
+
+  // Uses spacing tokens seen here:
+  // https://designsystem.digital.gov/design-tokens/spacing-units/#using-spacing-unit-tokens
+  .usa-card__header {
+    padding-top: units(1);
+    padding-bottom: units(1);
+  }
+
+  .usa-card__heading {
+    font-size: size('body', 'md');
+    text-align: center;
+  }
+}
```

Comment:
9. These create a new variation of card groups. Now you can use `usa-card-group--primary-bg` anywhere else on the site and have cards similar to the bottom of the homepage.